core/controllers/voice_artist.py

```diff
@@ -38,31 +39,56 @@ class AudioUploadHandler(base.BaseHandler):
     to the local datastore in dev).
     """
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'exploration_id': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'is_regex_matched',
+                    'regex_pattern': constants.ENTITY_ID_REGEX
+                }]
+            }
+        }
+    }
+
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'raw_audio_file': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'is_nonempty'
+                    }]
+                }
+            },
+            'filename': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'is_regex_matched',
+                        'regex_pattern': r'^[A-Za-z0-9-]+[.][a-z0-9]+$'
```

Comment:
whether we have filename pattern regex validation in develop? If not, are we sure that all other files in the storage follows this pattern?

Reply:
We don't have a specific filename pattern regex which is being used everywhere. Different upload handlers use their own raw regex string to validate the filename. Therefore, I used a similar pattern for audio upload.

Reply:
I'm not sure whether we should add such validation before ensuring all existing data follow this pattern.

@vojtechjelinek wdyt? In general do we prefer adding new validtaions through this PR?

Reply:
I think it depends on the type of validation, you're right that here we should be more careful and probably not use the regex.

Reply:
@vojtechjelinek @DubeySandeep Should I remove the validation then?

Reply:
Yeah

Reply:
@JayVivarekar Is this thread resolved?

Reply:
@DubeySandeep Yes I have removed the filename validation regex.